src/App.tsx

```diff
@@ -1,4 +1,5 @@
-import React from 'react';
+/* eslint-disable */
```

Comment:
This is a really bad idea

Reply:
omg I should have deleted this before pushing it :D I disabled it for not to be annoyed while dealing with other stuff but it looks it is not a good idea :D